### PR TITLE
update dev doc for samlAssertionNotOnOrAfter

### PR DESCRIPTION
**Pull request by miaopan-okta**
## Description:
- **What's changed?** 
- Added `samlAssertionLifetimeSeconds` descriptions
- **Is this PR related to a Monolith release?** 
- 2024.02.0

### Resolves:

* [OKTA-[675591](https://oktainc.atlassian.net/browse/OKTA-675591)](https://oktainc.atlassian.net/browse/OKTA-675591)

### CODE REVIEW

**Review by susanharper-okta — DISMISSED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/83df7a83dbe6c10f58b11957ab33efb35ad8d49d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2bc1fce6-56bf-4a41-bf5f-a399c7b2e0d3) | 69  | 60  | 65  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.fa04d3a9-9c97-42d7-8ee6-994e6f7d1a9a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/6ae63d0c2e51147fd66268030207d1318e7d3ccc/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d64ec1b0-4912-4a02-9166-cfe5c30f1bef) | 69  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.80a123c0-805a-405e-8aaf-57708e29c7d7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/2a502acb5485735e2686aea73cccd1ae51074068/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ba05f573-d549-4e93-b571-3ceb66b54597) | 68  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.3df23a41-1aee-49c0-b93e-2e16bffd1b9b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/820a510e92b251eb753de5d8b007f155dce5fff5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0f25219a-2d05-493d-a63f-449b04b78252) | 69  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.c2e09f15-20ee-43bf-be86-78a3c5a26c64?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/820a510e92b251eb753de5d8b007f155dce5fff5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e5444143-ba55-4bb1-bcc3-f68b9c3b1b5c) | 69  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.826fc0cb-d241-4f5f-83d6-e6f1919cefa1?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/444f88ab58b87e2693e4dbf5130c1bdd19ec7ce7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/98a11382-02f1-48a0-912f-cb603a3109b3) | 69  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.5f7248a7-5302-48dd-97da-a1e492adb4f3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 1]
```diff
@@ -978,34 +978,35 @@ Adds a SAML 2.0 application. This application is only available to the org that
...
+|--------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------| -------- |--------| --------------------------------------  |
+| allowMultipleAcsEndpoints                                                | Determines whether the app allows you to configure multiple ACS URIs                                                                                                                          | Boolean                                                        | FALSE    | FALSE  |                                           |
+| acsEndpoints                                                             | An array of ACS endpoints. You can configure a maximum of 100 endpoints.                                                                                                                      | Array of [ACS Endpoints](#acs-endpoint-object)                 | TRUE     | FALSE  |                                           |
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:984]:
```suggestion
| acsEndpoints                                                             | An array of ACS endpoints. You can configure a maximum of 100 endpoints                                                                                                                      | Array of [ACS Endpoints](#acs-endpoint-object)                 | TRUE     | FALSE  |                                           |
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 2]
```diff
@@ -978,34 +978,35 @@ Adds a SAML 2.0 application. This application is only available to the org that
...
+| audienceOverride                                                         | Overrides the `audience` setting                                                                                                                                                              | String                                                         | TRUE     | FALSE  |                                           |
+| authnContextClassRef                                                     | Identifies the SAML authentication context class for the assertion's authentication statement                                                                                                 | String                                                         | FALSE    | FALSE  |                                           |
+| defaultRelayState                                                        | Identifies a specific application resource in an IdP-initiated SSO scenario                                                                                                                   | String                                                         | TRUE     | FALSE  |                                           |
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:990]:
```suggestion
| defaultRelayState                                                        | Identifies a specific app resource in an IdP-initiated SSO scenario                                                                                                                   | String                                                         | TRUE     | FALSE  |                                           |
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 3]
```diff
@@ -978,34 +978,35 @@ Adds a SAML 2.0 application. This application is only available to the org that
...
+| honorForceAuthn                                                          | Prompts the user to re-authenticate if the SAML request has the `ForceAuthn` attribute set to `true`. If this property is `false`, the `ForceAuthn` attribute in the SAML request is ignored. | Boolean                                                        | FALSE    | FALSE  |                                           |
+| idpIssuer                                                                | SAML Issuer ID                                                                                                                                                                                | String                                                         | FALSE    | FALSE  |                                           |
+| inlineHooks                                                              | Associates the application with SAML inline hooks. See the [SAML assertion inline hook reference](/docs/reference/saml-hook/).                                                    | String                                                         | TRUE     | FALSE  |                                           |
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:996]:
```suggestion
| inlineHooks                                                              | Associates the app with SAML inline hooks. See the [SAML assertion inline hook reference](/docs/reference/saml-hook/).                                                    | String                                                         | TRUE     | FALSE  |                                           |
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 4]
```diff
@@ -978,34 +978,35 @@ Adds a SAML 2.0 application. This application is only available to the org that
...
+| requestCompressed                                                        | Determines whether the SAML request is expected to be compressed                                                                                                                      | Boolean                                                        | FALSE    | FALSE  |                                           |
+| responseSigned                                                           | Determines whether the SAML authentication response message is digitally signed by the IdP                                                                                            | Boolean                                                        | FALSE    | FALSE  |                                           |
+| samlAssertionLifetimeSeconds <ApiLifecycle access="ea" />                | Determines the SAML Application session lifetimes with Okta                                                                                                                                   | Integer                                                        | TRUE     | FALSE  |                                           |
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:1001]:
```suggestion
| samlAssertionLifetimeSeconds <ApiLifecycle access="ea" />                | Determines the SAML app session lifetimes with Okta                                                                                                                                   | Integer                                                        | TRUE     | FALSE  |                                           |
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 5]
```diff
@@ -978,34 +978,35 @@ Adds a SAML 2.0 application. This application is only available to the org that
...
+| signatureAlgorithm                                                       | Determines the signing algorithm used to digitally sign the SAML assertion and response                                                                                                       | String                                                         | FALSE    | FALSE  |                                           |
+| slo                                                                      | Determines if the application supports Single Logout                                                                                                                                          | [Single Logout](#single-logout-object)                         | TRUE     | FALSE  |
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:1003]:
```suggestion
| slo                                                                      | Determines if the app supports Single Logout                                                                                                                                          | [Single Logout](#single-logout-object)                         | TRUE     | FALSE  |
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 6]
```diff
@@ -978,34 +978,35 @@ Adds a SAML 2.0 application. This application is only available to the org that
...
+| participateSlo <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" /> | Determines if the application participates in Single Logout                                                                                                                                   | [Participate Single Logout](#participate-single-logout-object) | TRUE     | FALSE  |
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:1004]:
```suggestion
| participateSlo <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" /> | Determines if the app participates in Single Logout                                                                                                                                   | [Participate Single Logout](#participate-single-logout-object) | TRUE     | FALSE  |
```

**Review by barbaravo-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/1db183a7ef5965f671155223670d5afd71b3af09/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1c0b8afc-afc2-471e-a7ec-f351e3428102) | 69  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.87494ba1-6f42-4324-908b-188d1b510e43?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/71e4969d16ca94fe645cf0ca3f97b437b8ab8b9a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5af5faad-95c4-4a60-b5b8-3722bd1e85c6) | 69  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.ad5095b7-a911-45bc-99fe-2b035ce77b69?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/d7a979ff7b286a9a6b536a5e9a447fe20ed81fcd/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0ac2bf8e-5d62-456a-b57c-c09d211722eb) | 69  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e4358485-af7c-451c-98f6-0761544819ee?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/31793a8a6e314b95023c1514cebca9bdaf934230/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f4aac247-c41c-4aab-b1d9-4eab4b2e3345) | 69  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.8ada3623-32f2-459f-9786-0dadb7345959?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/53488bfb43e9fe20714c5a46c9885a744d3ff441/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4047a119-0bd4-45fc-a5b8-b2d55c1eaf5f) | 69  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e4829b63-08a7-4b6a-a5de-16a9dc1ac7a8?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/3e88eef75950314fddae0a9e2c12fd4964561c05/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fe4f9b4b-a546-4670-883a-1a200900f94a) | 69  | 60  | 66  | 71  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.9832c8ea-6509-4f10-9130-b5851bb26801?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**